### DISCUSSION TITLE
RGB-21 Schema: Do we need a NFT group ticker?

### DISCUSSION THREAD

**Discussion by UkolovaOlga** [LNP-BP/LNPBPs#77] [Q&A]
According to the dev call held on the 25.11.2020 https://github.com/LNP-BP/devcalls/commit/1fbb7e77bfbc6b11bb72c8bef544e5317e447dca 

This issue is a part of discussion https://github.com/LNP-BP/LNPBPs/issues/79

**Comment by sabina-sa** [comment]
I think we may need groups, for the following reasons:

1. The value of the collectionable items is often dependent on whether the collection is full (or close to being full) or not. The utility / pleasure that the collector receives increases sharply when he identifies and buys many items of a series. So, anything that attracts consumer’s attention to the fact that the item belongs to a series enhances the collector’s experience and increases the issuer’s profits. In addition, it matters for the programmable auctions - e.g., in the more complex Ebay auction software, the bids are contingent on whether other bids for items in the same series are already won or lost by the bidder. 

2. The ability to number (i.e. label with numbers) items within a group could be very useful for cataloguing purposes, and I think it is widely used in many industries that use catalogues. For example when collectible cards depict NBA players, it is very inconvenient to call them by last names or by long token IDs, so the convenient catalogue label of an item is usually something like NBA-145. It is easier to catalogue things within the system itself rather than rely on external catalogues.

Another topic which is very close to having groups, but perhaps it not exactly the same, is the continued proof of authorship of an anonymous author (an author who does not want to operate a website). Let’s say that someone creates a piece of art, which becomes controversially famous. Later, maybe even several years later, the same author would like to publish a sequel. Probably there should be a way to do so, without having to communicate and prove externally on a website that it is a sequel by the same author.

**Comment by fedsten** [comment]
I do not think we need a NFT group ticker. Tickers are used in exchange to shortly identify a fungible traded asset that traders can easily associate to its full name, but NFT due to their non fungible nature will probably the exchanged more in Ebay-like sited where, instead of tickers, names and descriptions are used to convey item identifying information to the user. I therefore believe that if a ticker field is added, it will likely not be used by the large majority of issuers, but the fact that still it is there will force wallet developers to support it in the user interface, making UX more complex and possibly causing confusion to the user as it may not be clear what the ticker information is supposed to represent.

Regarding the point raised by @sabina-sa, I have the following observations:

1. The information regarding the belonging to a specific series can be conveyed at social level (people just know it from external information), in the asset name or in the asset description. There is no need to have a specific field for something that may be relevant only for a small percentage of NFTs.
2. Similarly to the previous point, I believe that cataloguing information can just be stored in the description field as the are probably relevant just for a minority of use cases.
3. I think that proof of authorship from an anonymous author can be accomplished only with signature verification. If he desires to convey the information that a token is related to a previously issued one, he can still do it in the description field.

**Comment by sabina-sa** [comment]
I still feel it's fairly important, due to increased censorship in the internet "at the social level", to be able to convey as much useful information as possible inside RGB itself, without relying on external media. Also, I feel it is preferable when useful information is derived easily and automatically, so that authors do not rely on the wallet providers or final users to do that work. So, while group field may be unnecessary technically, they still might make sense socially and business-wise.

**Comment by rajarshimaitra** [comment]
I think I would also agree with @fedsten on the point that such affiliation of nonfungible assets is mostly at social layers, not on the asset itself. 

For example, The sequel of a controversial novel has its association with the identity of the author and the abstract ideas it contains within the words. There's nothing specific in the material of the object that signals this association. Also not having that signal engraved in the material doesn't impose any cost on us to extract it. Because its really not within the assets, but its in our head. So a ticker for that purpose would probably remain redundant. The same signalling can be done by engraving or using other descriptive fields of the assets. 

Also as collectible assets we can assume there won't be a very large number of them and fetching out this social signal from other descriptive fields would very trivial. Just like we fetch out social signals for controversial novels from the author's name or the title of the book. We don't need a separate "controversial" tag. It could have been added as trivially also. But the fact that we don't do that for books seems telling here imo.

**Comment by dr-orlovsky** [comment]
I am afraid here in discussion you are talking about different things.

@sabina-sa groups are already part of the RGB-21; the question is whether they should be identified by their long id (as today, bech32-string) or also (in addition) have a ticker of 3-8 chars like securities (w/o any guarantees on it's uniqueness, i.e. opening new possibilities for the scammers). In securities this is must have (since the industry works this way), but may be (and probably should be) avoided.

**Comment by sabina-sa** [comment]
Oh, I see. In that case, for economics it does not make a difference how the groups are technically identified - as long as the feature is present and easy to use. In fact, absence of uniqueness may be a good argument not to introduce a ticker.

**Comment by UkolovaOlga** [comment]
Feedback from @ctramount

Ethereum uses ticker-based "Collections" for their NFTs architecture. From my understanding, minted ERC721 tokens must be minted to a collection. They're nice for organizational purposes e.g., artists can mint all NFTs to their personal collection making their entire portfolio easy to query. However, collections create friction because they are expensive to mint ($10-30 USD at current gas levels).

It seems a large part of Rarible's success can be attributed to allowing issuers to mint to their collection "RARI" (https://app.rarible.com/create/erc721) with minimal gas fees, whereas Opensea required issuers to mint a collection (They've just recently launched a product to bypass this: https://opensea.io/storefronts-staging).

In sum, collections are worthwhile as long as they can be minted with minimal fees. An optimization feature that's not available on Ethereum could be to allow Collections to be shared among multiple issuers. An example in which this can be useful is with art collectives and collaborations.
